packages/vite/src/node/ssr/ssrManifestPlugin.ts

```diff
@@ -40,6 +44,51 @@ export function ssrManifestPlugin(config: ResolvedConfig): Plugin {
               })
             }
           }
+          if (chunk.code.indexOf(preloadMethod) > -1) {
```

Comment:
Should we just use `includes` here?

Reply:
There code refer to[importAnalysisBuild]( https://github.com/vitejs/vite/blob/main/packages/vite/src/node/plugins/importAnalysisBuild.ts#L252,)
do you think we should change it?

Reply:
IMO it is sometimes good to modernise code when currently refactoring it anyways. But if we have just two different opinions, let's bring in a third party.

/cc @patak-js 😸

Reply:
Yeah, i agree with modify judge condition andwe should keep them consistent.maybe we shoud change them together

Reply:
has updated @Shinigami92